test/scope-define-test.js

```diff
@@ -229,3 +231,26 @@ QUnit.test("this works everywhere (#45)", function(){
 	// this.foo works
 	QUnit.equal(scope.get("this.foo"),"bar");
 });
+
+QUnit.test("'this' and %context give the context", 2, function(){
```

Comment:
there are a bunch of tests using scope .read directly.  Please follow that pattern.